pkgs/development/python-modules/metakernel/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib,
+  buildPythonPackage,
+  fetchPypi,
+  ipykernel,
+  isPy27,
+  mock,
+  pytest
+}:
+
+buildPythonPackage rec {
+  pname = "metakernel";
+  version = "0.27.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0aqq9zil6h7kxsg3v2008nr6lv47qvcsash8qzmi1xh6r4x606zy";
+  };
+
+  buildInputs = [ipykernel] ++ lib.optional isPy27 [mock pytest];
+  propagatedBuildInputs = [ipykernel];
+
+  # Tests hang, so disable
```

Comment:
Why does it hang?

Reply:
Not sure, I see several errors and then no more output:

```
writing manifest file 'metakernel.egg-info/SOURCES.txt'
running build_ext
magic (unittest.loader._FailedTest) ... ERROR
process_metakernel (unittest.loader._FailedTest) ... ERROR
parser (unittest.loader._FailedTest) ... ERROR
_metakernel (unittest.loader._FailedTest) ... ERROR
```